pkg/services/ngalert/notifier/channels/templates/default.tmpl

```diff
@@ -0,0 +1,219 @@
+{{ define "__alertmanager" }}Alertmanager{{ end }}
```

Comment:
This file is taken from Prometheus Alertmanager. I will remove unwanted stuff in the future PRs once we have all notification channels in.

Reply:
Can we create an issue for it please?

Reply:
I will create one after I merge this PR